client/components/App.jsx

```diff
@@ -1,8 +1,12 @@
 import React from 'react'
+import Header from './Header'
```

Comment:
This will be imported into Dashboard please remove changes to app.

Reply:
Thanks sis removed through the latest commit called : "https://github.com/kahikatea-2019/MOA/pull/32/commits/7485c631f07a22df4f1706a350b1ad85dc0b3f17 "